### DISCUSSION TITLE
Column Stopped Working

### DISCUSSION THREAD

**Discussion by bland-industries** [strayer/taira-keyboard#10] [Q&A]
First, thanks for this project, I have enjoyed my keyboard for many months now.

Second, Just recently a column stopped working. I tried re-soldering what I think were the connections to the board and that did nothing. I am here asking for advice on how to trouble shoot or fix. It is the second column (going left to right) on the left side.

**Comment by madushan1000** [comment]
I had a similar issue, It was a bad solder joint in my controller(nrfmicro). test if all the gpio pins work.

**Comment by strayer** [comment]
Hey! Sorry for not responding. Funnily enough I had a similar issue with the number row not working on my second half for a while. In my case it was also a bad solder joint that was fixed by resoldering the headers for the nice!nano. Before that I had to wiggle the nice!nano a bit to "reconnect" the solder joints.

Since you already tried resoldering: You can also test this by shorting the relevant GPIO pins on the nice!nano with wires, pliers or something else conducting. Just be extra sure that you short the correct GPIO pins and don't fry the nice!nanos! Been there… 🙈 Check the schematics which GPIO pins would be shorted by the keyboard matrix for the column you are experiencing issues with - when shorting the GPIOs manually the system the keyboard is paired to should receive the key press. If not I'd assume something might be wrong with the nice!nano.
